kieserver/image.yaml

```diff
@@ -199,7 +199,31 @@ envs:
     description: "Disable Maven pull dependencies for immutable KIE Server configurations for S2I and pre built kjars. Useful for pre-compiled kjar."
   - name: "KIE_SERVER_DISABLE_KC_VERIFICATION"
     example: "true"
-    description: "Disable Maven KIE Jar verification. It is recommended to test the kjar manually before disabling this verification."
+    description: "Disable Maven KIE Jar verification. It is recommended to test the kjar manually before disabling this verification."  
+  - name: "KIE_SERVER_KAFKA_EXT_ENABLED"
+    example: "true"
+    description: "Setting with true enable the others KIE_KAFKA_EXT environments used by Kafka, this env will be used by system prop org.kie.kafka.server.ext.disable with the opposite value"
+  - name: "KIE_SERVER_KAFKA_EXT_TOPICS"
```

Comment:
maybe improve this to topic_name=topic_name.
see my comments on jboss-container-images/jboss-kie-modules#446

Reply:
Done